docs/releasing.md

```diff
@@ -1,38 +1,106 @@
+
+- Tag pushing
+- GitHub Release publishing
+
+These permissions are implicit for the org admins and repository admins.
+Release team member gets his/her permissions via `metal3-release-team`
+membership. This GitHub team has the required permissions in each repository
+required to release BMO. Adding person to the team gives him/her the necessary
+rights  in all relevant repositories in the organization. Individual persons
+should not be given permissions directly.
+
+## Process
+
+BMO uses [semantic versioning](https://semver.org).
+
+### Repository setup
+
+Clone the repository:
+`git clone git@github.com:metal3-io/baremetal-operator`
+
+or if using existing repository, verify your intended remote is set to
+`metal3-io`: `git remote -v`. For this document, we assume it is `origin`.
+
+### Tags
+
+BMO does not have release branches. Release is created by tagging a commit off
+the main branch. First we create a primary release tag, that triggers release
+note creation and image building processes.
+
+- Create a signed, annotated tag with: `git tag -s -a v0.x.y -m v0.x.y`
+- Push the tags to the GitHub repository: `git push origin v0.x.y`
+
+This triggers two things:
+
+- GitHub action workflow for automated release process creates a draft release
+  in GitHub repository with correct content, comparing the pushed tag to
+  previous tag
+- Quay starts building release image with the release tag
+
+We also need to create one or more tags for the Go modules ecosystem:
+
+- For any subdirectory with `go.mod` in it (excluding `hack/tools`), create
+  another Git tag with directory prefix, ie.
+  `git tag -s -a api/v1.x.y -m api/v1.x.y`.
+  For BMO, these directories are `api` and `pkg/hardwareutils`.
+
+### Release artifacts
+
+We need to verify all release artifacts are correctly built or generated by
+the release workflow. For a release, we should have the following artifacts:
+
+Git tags pushed:
+
+- Primary release tag: `v0.x.y`
+- Go module tags: `api/v0.x.y` and `pkg/hardwareutils/v0.x.y`
+
+Container images built and tagged at Quay registry:
+
+- [baremetal-operator:v0.x.y](https://quay.io/repository/metal3-io/baremetal-operator?tab=tags)
+
+BMO release in GitHub will only contain source code as artifact.
+
+### Release notes
+
+Next step is to clean up the release note manually.
+
+- Check for duplicates, reverts, and incorrect classifications of PRs, and
+  whatever release creation tagged to be manually checked.
+- For any superseded PRs (like CAPI uplifts, or commit revertions) that provide
+  no value to the release, create a summary line in `Superseded` section with
+  the PR ids and summary title. This way the changes are acknowledged to be part
+  of the release, but not overwhelming the important changes contained by the
+  release.
+- If the release you're making isn't the latest release (new
+  minor or major, or new patch release in the latest minor release),
+  uncheck the box for latest release.
```

Comment:
same comment as in other similar PR: https://github.com/metal3-io/cluster-api-provider-metal3/pull/916#discussion_r1175772191

Reply:
Fixed, is it clearer now?